crates/chainio/clients/src/lib.rs

```diff
@@ -0,0 +1,139 @@
+//! Integration test for rewards utilities
+#[cfg(test)]
+pub mod tests {
+
+    use alloy::primitives::{aliases::U96, U256};
+    use eigen_common::{get_provider, get_signer};
+    use eigen_testing_utils::{
+        anvil::start_anvil_container,
+        anvil_constants::{
+            get_erc20_mock_strategy, get_rewards_coordinator_address, FIRST_ADDRESS,
+            FIRST_PRIVATE_KEY,
+        },
+        chain_clients::{
+            build_avs_registry_chain_writer, build_el_chain_reader, new_claim, new_test_writer,
+        },
+        transaction::wait_transaction,
+    };
+    use eigen_utils::slashing::{
+        core::irewardscoordinator::IRewardsCoordinator,
+        middleware::servicemanagerbase::IRewardsCoordinatorTypes::{
+            RewardsSubmission, StrategyAndMultiplier,
+        },
+        sdk::mockerc20::MockERC20,
+    };
+
+    #[tokio::test]
+    async fn test_process_claim() {
+        let (_container, http_endpoint, _ws_endpoint) = start_anvil_container().await;
+        let signer = get_signer(FIRST_PRIVATE_KEY, &http_endpoint);
```

Comment:
We can move this private key to a variable too.

```suggestion
        let rewards_initiator_sk = FIRST_PRIVATE_KEY;
        let signer = get_signer(rewards_initiator_sk, &http_endpoint);
```

The idea is to know which different roles exist in the example. Here we have the "rewards initiator" with permission to update the distributions root, and the claimer, with permission to process the claims created by `new_claim`.

Reply:
Done in [b3df1da](https://github.com/Layr-Labs/eigensdk-rs/pull/404/commits/b3df1da8f59774d79545c03acfa6c54d7e53973a)